Error in packaging

There's an error in the packaging right now that just recently came about.

```python
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/hexrdgui-package/bin/anaconda", line 6, in <module>
    from binstar_client.scripts.cli import main
  File "/usr/share/miniconda/envs/hexrdgui-package/lib/python3.7/site-packages/binstar_client/__init__.py", line 26, in <module>
    from .requests_ext import NullAuth
  File "/usr/share/miniconda/envs/hexrdgui-package/lib/python3.7/site-packages/binstar_client/requests_ext.py", line 11, in <module>
    from urllib3.filepost import choose_boundary, iter_fields
ImportError: cannot import name 'iter_fields' from 'urllib3.filepost' (/usr/share/miniconda/envs/hexrdgui-package/lib/python3.7/site-packages/urllib3/filepost.py)
Error: Process completed with exit code 1.
```

--- a/packaging/environment.yml
+++ b/packaging/environment.yml
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.7
+  - python=3.8
   - coloredlogs
   - conda
   - cmake

